src/main/resources/swagger.api/mod-fqm-manager.yaml

```diff
@@ -62,7 +62,23 @@ paths:
           $ref: '#/components/responses/badRequestResponse'
         '500':
           $ref: '#/components/responses/internalServerErrorResponse'
-
+  /fqm/version:
+    get:
+      operationId: getQueryVersion
+      tags:
+        - queryVersion
+      description: Get the version of the fqm query.
+      responses:
+        '200':
+          description: 'Version of the fqm query'
```

Comment:
Small thing, but I'd update these to no longer refer to query, since the version covers more than just queries (queries are the things that get updated, based on the FQM version, so the version is associated with FQM and not the query)